.eslintrc

```diff
@@ -1,3 +1,27 @@
 {
-  "extends": ["vtex-react-app"]
+  "parser": "babel-eslint",
```

Comment:
Why update this file?

Reply:
This is the IO linting rules. 
Soon it will be automatic and it will not be up to the user anymore to define his linting rules. So better get used to it ASAP.